CONTRIBUTING.txt

```diff
@@ -173,48 +173,8 @@ Once you've fixed all merge conflicts, do::
 Build environment setup
 -----------------------
 
-Once you've cloned your fork of the scikit-image repository,
-you should set up a Python development environment tailored for scikit-image.
-You may choose the environment manager of your choice.
-Here we provide instructions for two popular environment managers:
-``venv`` (pip based) and ``conda`` (Anaconda or Miniconda).
-
-venv
```

Comment:
These more elaborate instructions were included due to new contributors having a hard time getting started.


I think the crazy `cat` command is too much, but I think the rest should be kept.

From creating the environment to testing.

Reply:
Your comments are appreciated, @hmaarrfk.

I'm unclear on the `cat` -- I started my branch from yours (`hmaarrfk:beginner_install_instruction`), changing only INSTALL.rst.  Is there something else I should change?

Reply:
Right, we had some more elaborate development instructions in the CONTRIBUTING.txt file.

These included the creation of a venv.

I think it is necessary to include those for first time contributors.